server/models/Polls.js

```diff
@@ -24,8 +24,8 @@ const PollSchema = new mongoose.Schema({
     type: String,
     require: true,
   },
-  votesForUrl1: [String],
-  votesForUrl2: [String],
+  votesForUrl1: [mongoose.Types.ObjectId],
```

Comment:
This actually need to look like this:
`[{ type : ObjectId, ref: 'User' }],`

Reply:
are you sure ? isn't it mongo is now expection 
```
[{type:'absajhbjhsfdgjhsfd' , ref:'Something'}] 
```
???

Reply:
No, when you updated the field, it will expect something like this: poll.votesForUrl1 = [userId1,userId2], etc